apps/rush-lib/src/data/VersionPolicy.ts

```diff
@@ -44,10 +44,13 @@ export abstract class VersionPolicy {
   public static load(versionPolicyJson: IVersionPolicyJson): VersionPolicy {
     const definition: VersionPolicyDefinitionName = VersionPolicyDefinitionName[versionPolicyJson.definitionName];
     if (definition === VersionPolicyDefinitionName.lockStepVersion) {
+       // tslint:disable-next-line:no-use-before-declare
       return new LockStepVersionPolicy(versionPolicyJson as ILockStepVersionJson);
     } else if (definition === VersionPolicyDefinitionName.individualVersion) {
+      // tslint:disable-next-line:no-use-before-declare
```

Comment:
Out of scope.

---
In reply to: [129394167](https://github.com/Microsoft/web-build-tools/pull/276#discussion_r129394167) [](ancestors = 129394167)